osu.Game/Beatmaps/BeatmapSetInfo.cs

```diff
@@ -85,7 +86,7 @@ public bool Equals(BeatmapSetInfo other)
 
         #region Implementation of IHasOnlineID
 
-        public int OnlineID => OnlineBeatmapSetID ?? -1;
+        int IHasOnlineID<int>.OnlineID => OnlineID ?? -1;
```

Comment:
Tiny thing but `BeatmapInfo` delegates to the backing field, but this delegates to the property getter. Would probably prefer to unify handling.

Reply:
Agree with changing to use the property getter.